### PR TITLE
LoW: Make gold carry-over consistent with objectives

### LINKED ISSUE #7230: Legend of Wesmere S11 and S14 gold carry-over inconsistency
### Language

All

### Source

Battle of the Book, Campaign Legends of Wesmere

### Original Text

40% carried gold

### Proposed Change

80% carried gold

Note: If it's not a typo, and it's an error in the mission objectives and it really should be 40%, then change the objectives

### PR DESCRIPTION
**Pull request by Wedge009**
Resolves #7230

### CODE REVIEW

File: data/campaigns/Legend_of_Wesmere/scenarios/chapter3/11_Battle_of_the_Book.cfg [hunk 1]
```diff
@@ -284,6 +284,7 @@
 #else
             bonus=yes
 #endif
+            carryover_percentage=40
```
Comment (knyghtmare) [inline data/campaigns/Legend_of_Wesmere/scenarios/chapter3/11_Battle_of_the_Book.cfg:287]: I think the carryover here should be 80 and not 40. Do you even know what the next scenario is?
2 cutscenes and then the large-scale orc vs elf & human battle. I dont think players would even approve of this...

File: data/campaigns/Legend_of_Wesmere/scenarios/chapter4/14_Human_Alliance.cfg [hunk 2]
```diff
@@ -586,6 +586,7 @@ Chapter Four"
             result=victory
             bonus=no
             save=no
+            carryover_percentage=40
```
Comment (knyghtmare) [inline data/campaigns/Legend_of_Wesmere/scenarios/chapter4/14_Human_Alliance.cfg:589]: I really think the balance adjustments should be playtested to ensure it's at least passable.

**Review by knyghtmare — CHANGES_REQUESTED** [review]

File: data/campaigns/Legend_of_Wesmere/scenarios/chapter3/11_Battle_of_the_Book.cfg (hunk 1 above)
Comment (Wedge009) [reply data/campaigns/Legend_of_Wesmere/scenarios/chapter3/11_Battle_of_the_Book.cfg:287]: 80 is the default. Yes, I saw that the next scenario is Human Alliance. The question is what the intention is for converting all the campaigns - including this one - to use 40% by default. Why make it 80 if all the others are 40?

File: data/campaigns/Legend_of_Wesmere/scenarios/chapter4/14_Human_Alliance.cfg (hunk 2 above)
Comment (Wedge009) [reply data/campaigns/Legend_of_Wesmere/scenarios/chapter4/14_Human_Alliance.cfg:589]: Shouldn't that sort of thing have been done when everything was converted to 40% in the first place?

**Comment by Wedge009** [comment]
A reminder: this change isn't intended to be a 'balance' change. It's intended to make things consistent for what is already there, as highlighted by #7230.

The conversion to 40% was done by @nemaara in dbaeeea57e49d8f1fe07d3e9e149dca4e119141e, present since 1.15.0. It appears starting gold was already adjusted to accommodate the change in carry-over. So, to me, it looks like where carry-over remained at the default 80%, it's actually an oversight.

**Comment by stevecotton** [comment]
If the bug was present when it was playtested (which I assume it was), then it seems necessary to assume that @nemaara tested it with the bug present, and balanced based on getting there with 80% gold.

**Comment by Wedge009** [comment]
Hence why mentioned @nemaara, for input.

**Comment by nemaara** [comment]
I can't remember why I missed this one but it'll need playtesting to fix the values. We can't blindly make it 40% without changing the gold values in the next scenario.

**Comment by Wedge009** [comment]
Who is qualified for play-testing, then? I don't consider myself a suitable candidate. In the case that 80%->40% proves to be too difficult, is it just a matter of increasing the starting gold for the subsequent scenarios? I observe this was already done in the original conversion to 40%.

**Comment by Wedge009** [comment]
I just came back from a short break - I'll update with your suggestions but will leave it open for now.

**Comment by nemaara** [comment]
Can we leave this open for a week to see if more comments come in? After a week or so we can consider merging.

**Comment by Wedge009** [comment]
I didn't set a specific time-frame, but yes, I left it open for further input.

**Comment by Wedge009** [comment]
Has been a bit more than a week now and 1.17.12 has been tagged already. Are there any outstanding objections to this?

**Comment by nemaara** [comment]
Let's say give 2 more days then merge?

**Comment by stevecotton** [comment]
Should this be backported?

**Comment by Wedge009** [comment]
See above, since this became a balance change I suggested not to back-port.

**Pull request merged**